docs/developersguide/testing-practices.rst

```diff
@@ -0,0 +1,72 @@
+Testing practices
+=================
+
+It is a good practice to add test cases to each Invenio module. Tests are
+normally organized in:
+
+1. Unit functional testing: test cases run against the Flask app defined in the
```

Comment:
Good point. The thing is that, IMHO, the tests I see in the `unit` folder, are not 100% unit test, where you test one specific function/method input and outputs. What I see there, are functional tests where you run a Flask app and you test a feature, but without UI.
Am I wrong?

So, again IMHO, those are not unit tests.
Any opinion @switowski and @lnielsen?